lib/modules/dosomething/dosomething_api/resources/reportback_item_resource.inc

```diff
@@ -158,7 +158,58 @@ function _reportback_item_resource_index($campaigns, $exclude, $status, $count,
 
   if (variable_get('rogue_collection', FALSE)) {
     $client = dosomething_rogue_client('v1');
-    $response = $client->getAllReportbacks($inputs);
+
+    // Get the query params to see if we are populating client or server side gallery portion.
+    // If page param is present, we are populating "view more" section of gallery (client side).
+    $query_parts = explode("&", $_SERVER['REQUEST_URI']);
+    $query_params = [];
```

Comment:
omg I am a dummy! I swear I tried to do that and it didn't work but just tried again and it did. So much better!!